src/containers/Media/Media.js

```diff
@@ -25,7 +25,17 @@ export default class extends Component {
   render() {
     return (
       <div className={style.block}>
-        <img className={style.image} alt={this.props.image} src={this.props.media.selectedImage || this.props.image} />
+        <picture className={style.image}>
+          {/*<!--[if IE 9]><video style="display: none;"><![endif]-->*/}
+          <source alt={this.props.image} srcSet={this.props.media.selectedImage || this.props.image[0] + ', ' + this.props.image[1] + ' 2x'} media="(min-width: 400px)" />
+          {/*<!--[if IE 9]></video><![endif]-->*/}
+          <img
+            alt={this.props.image}
+            className={style.image}
+            srcSet={this.props.media.selectedImage || this.props.image[0] + ', ' + this.props.image[1] + ' 2x'}
+            src={this.props.media.selectedImage || this.props.image[0]}
```

Comment:
Setting the `src` attribute on this `img` will result in double requests for browsers without native picture support (IE, all but latest Safari). We might want to leave this off for as long as we polyfill picture.